tests/wallet_main_screen/test_send.py

```diff
@@ -0,0 +1,27 @@
+import allure
+import pytest
+from allure_commons._allure import step
+
+import configs
+import constants
+from constants.wallet import ARBITRUM_ADDRESS
+from gui.components.signing_phrase_popup import SigningPhrasePopup
+from gui.components.wallet.authenticate_popup import AuthenticatePopup
+from gui.main_window import MainWindow
+
+
+@allure.testcase('https://ethstatus.testrail.net/index.php?/cases/view/704527', 'Wallet.Send.')
+@pytest.mark.case(704527)
+@pytest.mark.parametrize('user_data', [configs.testpath.TEST_USER_DATA / 'squisher'])
+def test_wallet_main_screen_send_null_eth(main_screen: MainWindow):
```

Comment:
```suggestion
def test_wallet_main_screen_send_0_eth(main_screen: MainWindow):
```
None and 0 are different values. We are not sending None, we are sending 0